products/iam/examples/ansible/service_account.yaml

```diff
@@ -11,12 +11,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 --- !ruby/object:Provider::Ansible::Example
-vars:
-  sa_name: "sa-{{ 100000 | random }}@graphite-playground.google.com.iam.gserviceaccount.com"
 task: !ruby/object:Provider::Ansible::Task
   name: gcp_iam_service_account
   code:
-    name: "{{ sa_name }}"
+    # We need a constant random number.
```

Comment:
"Constant random number"? ;-) How about something like "The randomized service account name suffix must remain constant during test invocation" or some such

Reply:
That's much better. I'll make that fix.